assets/js/hooks/session.js

```diff
@@ -447,6 +447,12 @@ const Session = {
       return;
     }
 
+    // If a cell output is clicked, keep the insert mode as is
+    // and remove the focus from editor
+    if (event.target.closest(`[data-el-output]`)) {
+      return;
+    }
```

Comment:
In insert mode the editor is always focused, if it blurs it immediately focuses itself. Unfortunately we can't do it this way, because it breaks the case where we are in insert mode and want to select text in the output.

The problem here comes from the fact that exiting insert mode hides stuff, it's the same with markdown cell preview. One thing we could do is keep insert mode when clicking on the copy-to-clipboard button, this way the user can get setup cell output. It's definitely not perfect, but I can't think of anything else for now.

Reply:
I could improve this check and only returns when `focusableId` is `setup`. This situation occours when you evaluate the `Mix.install/1` or other setup stuff that uses the same code block, and it return some errors, so you want to scroll the message to see what happened.

But when you click on setup's output, it hides and you can't get the message error or anything inside that. The same happens if you click the `copy-to-clipboard`.

Reply:
Also, I'm still with the focus on the editor, but I ain't hiding the setup's output anymore. Maybe I could change the focus to the output so the text can be selected and copied, like the other cell outputs.